src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ data }) {
+  return (
+    <div>
+      {data}
+    </div>
+  );
+}
+
+Display.propTypes = {
+  data: PropTypes.string.isRequired,
+};
```

Comment:
This component should render `data` of type string with default value of '0'. Please add the required validation.